### ISSUE TITLE
Color contrast for icons and (en/fr) text needs to be stronger

### ISSUE THREAD

**Issue by piet0024** [gctools-outilsgc/gcconnex#943]
The site icons and the (en/fr) text that appears beside content that is in both languages needs to be darker to pass color contrast requirements for WCAG 2.0 AA

@CoolColin

**Comment by CoolColin** [comment]
WCAG 2.0 1.4.3 Contrast (Minimum): The visual presentation of text and images of text has a contrast ratio of at least 4.5:1, except for the following: (Level AA)
Large Text: Large-scale text and images of large-scale text have a contrast ratio of at least 3:1;
Incidental: Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.
Logotypes: Text that is part of a logo or brand name has no minimum contrast requirement.
